### DISCUSSION TITLE
Bouquets et playlist de données 🧺

### DISCUSSION THREAD

**Discussion by agarrone** [datagouv/data.gouv.fr#1199] [Ideas]
Plusieurs équipes ont fait remonter le besoin ou utilise le concept de bouquets de données. 
Cette discussion vise à permettre à chaque équipe de clarifier et spécifier son besoin pour permettre à data.gouv.fr d'y répondre au mieux.

**Comment by Haelle** [comment]
### Côte Pôle API - DINUM

**L'objectif final est la découvrabilité des cas d'usage d'api.gouv.fr**

Le Pôle API aujourd'hui a fortement évolué pour se concentrer sur la qualification, le référencement et la découvrabilité de "cas d'usages".

#### Un bouquet vs un cas d'usage ? 

Un bouquet est un groupe de sources de données sans "vrai" travail de mise en contexte métier. Là où un cas d'usage va plus loin et met en contexte les données d'un point de vue métier : 
- description exhaustive du besoin métier ;
- identification des champs utile dans chaque source de données (API & fichier) ;
- rappel du cadre juridique applicable ;
- limites du bouquet de données (périmètre : technique, géographique, métier) ;
- FAQ ;
- illustration de "réutilisations" (pour utiliser le vocabulaire data.gouv.fr ;) ), sous forme de maquettes pour illustrer comment la données peut être réutilisée ;
- documentation orientée pour les équipes métier (ie non-tech)
- documentation oreintée asussi pour les équipes tech
